### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ppf2**
Clarify that the extension is no longer maintained and recommend the open connectors framework.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ This extension syncs data from Google's [Cloud Firestore](https://firebase.googl
 
 Out-of-the-box, Cloud Firestore provides no mechanism for full-text search on data. Syncing your Cloud Firestore data to Elastic App Search not only gives you a mechanism for full-text search on your data, it also lets you enjoy App Search's powerful relevance tuning features and search analytics data.
 
+**NOTE:** This extension is no longer maintained.  We encourage the community use the open and supported [connectors framework](https://www.elastic.co/guide/en/enterprise-search/current/connectors.html#connectors-overview-framework) for Google Cloud Firestore integration development.
```
Comment (poremland) [inline README.md:7]: @ppf2 - as written this makes it appear that the connector framework already supports Firestore as a connector, which it doesn't. I'd suggest something like:
> We encourage the community to use the open and supported [connectors framework](https://www.elastic.co/guide/en/enterprise-search/current/connectors.html#connectors-overview-framework) to build an Elasticsearch connector for integration Google Cloud Firestore.

**Review by poremland — CHANGES_REQUESTED** [review]
Can you clarify such that it doesn't appear that we're eluding to the connector already existing.

File: README.md (hunk 1 above)
Comment (ppf2) [reply README.md]: Thanks and updated :)

**Review by poremland — APPROVED** [review]

**Comment by ppf2** [comment]
@JasonStoltz Can you help merge this README file change?  Thanks!

**Review by JasonStoltz — APPROVED** [review]

**Pull request merged**